src/test/java/org/broadinstitute/dsm/careevolve/Covid19OrderRegistrarTest.java

```diff
@@ -1,103 +0,0 @@
-package org.broadinstitute.dsm.careevolve;
-
-import com.google.gson.Gson;
-import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
-import com.typesafe.config.Config;
-import com.typesafe.config.ConfigFactory;
-import org.apache.commons.dbcp2.PoolableConnection;
-import org.apache.commons.dbcp2.PoolingDataSource;
-import org.broadinstitute.ddp.db.TransactionWrapper;
-import org.broadinstitute.dsm.cf.CFUtil;
-import org.broadinstitute.dsm.db.DDPInstance;
-import org.broadinstitute.dsm.db.DdpKit;
-import org.broadinstitute.dsm.statics.ApplicationConfigConstants;
-import org.broadinstitute.dsm.statics.DBConstants;
-import org.broadinstitute.dsm.util.ElasticSearchUtil;
-import org.elasticsearch.client.RestHighLevelClient;
-import org.junit.BeforeClass;
-import org.junit.Ignore;
-import org.junit.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.sql.Connection;
-import java.text.SimpleDateFormat;
-import java.time.Instant;
-import java.util.Map;
-
-
-public class Covid19OrderRegistrarTest {
-
-    private static final Logger logger = LoggerFactory.getLogger(Covid19OrderRegistrarTest.class);
-
-    private static Authentication auth;
-
-    private static String careEvolveOrderEndpoint;
-
-    private static String  careEvolveAccount;
-
-    private static Config cfg;
-
-    private static Provider provider;
-
-    @BeforeClass
-    public static void beforeClass() throws Exception {
-        cfg = ConfigFactory.load();
-        TransactionWrapper.init(20, cfg.getString("portal.dbUrl"), cfg, true);
-
-        // todo pull this out to a file, refresh from secret manager
-
-    }
-
-    @Ignore
-    @Test
-    public void testOrderForParticipant() throws Exception {
```

Comment:
The only thing going on here was placing an order, so in the spirit of not doing stuff like this in a "test", it's gone!